src/cascadia/PublicTerminalCore/HwndTerminal.cpp

```diff
@@ -349,7 +349,7 @@ IRawElementProviderSimple* HwndTerminal::_GetUiaProvider() noexcept
     return _uiaProvider.Get();
 }
 
-HRESULT HwndTerminal::Refresh(const SIZE windowSize, _Out_ COORD* dimensions)
+HRESULT HwndTerminal::Refresh(_In_ const SIZE windowSize, _In_ bool autoFit, _Out_ COORD* dimensions)

```

Comment:
Hey on these methods that take `autoFit` as a param, could we add a doc comment that describes what true/false does? Because reading the code, I'm not immediately sure if true or false means "resize the buffer as well as the renderer size"